tdrs-backend/tdpservice/parsers/row_schema.py

```diff
@@ -101,6 +104,12 @@ def run_preparsing_validators(self, line, generate_error):
 
         return is_valid, errors
 
+    def __set_skip_partial_dup_check(self, field, value):
+        # This technically could be overriden if this applies to other fields, but as of right now, FAMILY_AFFILIATION
+        # is the only field that has `can_skip_partial` set to True.
+        if field.can_skip_partial:
+            self.should_skip_partial_dup = value in self.skip_values
```

Comment:
@jtimpe let me know what you think of this solution. Not sure that I love it. I don't love it because now it seems that the hash generation itself should be moved up into the schema (which it could be). Im not sure it makes sense for the schema to hold that type of data though.

Reply:
perhaps the schema could store a function that generates a hash from the raw line or the partial hash from the parsed values (or maybe two functions `full_hash` and `partial_hash`). though i am seeing this as a bit of a rabbit hole. maybe we're better off coming back and refactoring in a separate ticket